src/WebServer_JSON.ino

```diff
@@ -16,10 +16,11 @@ void handle_rawval()
     htmlData += F("ERROR: TaskNr not valid!\n");
   }
   const taskVarIndex_t valNr    = getFormItemInt(F("valnr"), INVALID_TASKVAR_INDEX);
-  const bool valueNumberValid = validTaskVarIndex(valNr);
-  if (!valueNumberValid)
+  bool valueNumberValid = true;
+  if (valNr != INVALID_TASKVAR_INDEX && !validTaskVarIndex(valNr))
```

Comment:
See the code where it is checked:
```c++
taskVarIndex_t INVALID_TASKVAR_INDEX = VARS_PER_TASK;

bool validTaskVarIndex(taskVarIndex_t index) {
  return index < VARS_PER_TASK;
}
```
So this check may fail when `valNr == INVALID_TASKVAR_INDEX`